src/doc/trpl/compound-data-types.md

```diff
@@ -47,7 +47,8 @@ This pattern is very powerful, and we'll see it repeated more later.
 
 There are also a few things you can do with a tuple as a whole, without
 destructuring. You can assign one tuple into another, if they have the same
-arity and contained types.
+contained types and arity. Tuples have the same arity when they have the same
```

Comment:
On #21568 there is a big discussion on this topic. 

As for this commit, I think it is kind of a middle ground. And a concrete improvement.

But I guess length would also work